Gems/ScriptCanvas/Code/Editor/View/Windows/MainWindow.cpp

```diff
@@ -1125,10 +1125,10 @@ namespace ScriptCanvasEditor
             return AZ::Success(outTabIndex);
         }
 
-        auto result = LoadFromFile(fileAssetId.Path().c_str());
+        auto result = LoadFromFile(fileAssetId.AbsolutePath().c_str());
```

Comment:
There are some changes in this function to `AbsolutePath` in this function are deliberate, and they fix the Open in SC Editor function.